FinniversKit/Sources/Components/JobKeyInfo/Subviews/JobKeyInfoView+InfoPairView.swift

```diff
@@ -0,0 +1,48 @@
+import Foundation
+
+extension JobKeyInfoView {
+    class InfoPairView: UIView {
+
+        // MARK: - Private properties
+
+        private lazy var stackView: UIStackView = {
+            let stackView = UIStackView(axis: .horizontal, spacing: .spacingS, withAutoLayout: true)
+            stackView.alignment = .top
+            return stackView
+        }()
+
+        private lazy var titleLabel: Label = {
+            let label = Label(style: .bodyStrong, withAutoLayout: true)
+            label.numberOfLines = 0
+            return label
+        }()
+
+        private lazy var valueLabel: Label = {
+            let label = Label(style: .body, withAutoLayout: true)
+            label.textAlignment = .right
+            label.numberOfLines = 0
+            return label
+        }()
+
+        // MARK: - Init
+
+        init(infoPair: InfoPair, withAutoLayout: Bool) {
+            super.init(frame: .zero)
+            translatesAutoresizingMaskIntoConstraints = !withAutoLayout
+            setup(infoPair: infoPair)
+        }
+
+        required init?(coder: NSCoder) { fatalError() }
```

Comment:
Just to follow same style, but not necessary: "fatalError("init(coder:) has not been implemented")"